auth/token.go

```diff
@@ -19,9 +23,12 @@ const (
 	// TokenValidityInHours represents the token validity; used to set token expiry
 	TokenValidityInHours = 10
 
-	// TokenSigningKey is used for signing the token
-	// FIXME: this should be fetched from store
-	TokenSigningKey = "ccn$!~56"
+	// this is the datastore key, not the actual bytes used for signing.
+	// since changes are not handled by an abstraction layer, construct the path manually
+	tokenSigningKey = types.AuthProxyDir + "/token_signing_key"
```

Comment:
This the place we maintain all the datastore paths https://github.com/contiv/auth_proxy/blob/master/db/constants.go. And, we use `GetPath` to get the absolute datastore path like here https://github.com/contiv/auth_proxy/blob/master/db/local.go#L61

I recommend to follow the same style.